src/common/authRoleStatus.component.tsx

```diff
@@ -0,0 +1,67 @@
+import { IconButton, Paper, Tooltip, Typography } from '@mui/material';
+import Grid from '@mui/material/Grid2';
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import { getUserRole } from '../parseTokens';
+
+function AuthRoleStatus() {
+  const role = getUserRole();
+
+  return (
+    role && (
+      <Paper
+        square
+        sx={{
+          backgroundColor: '#FFA500',
+          display: 'flex',
+          flexDirection: 'column',
+          paddingLeft: 0,
+          paddingRight: 5,
+          justifyContent: 'center',
+        }}
+      >
+        <Grid
+          container
+          direction="row"
+          alignItems="center"
+          justifyContent="center"
+          aria-label="admin-status-info"
+        >
+          <Grid>
+            <Tooltip
+              title={
+                <h4>
+                  As a privileged user, you can create/edit settings, and bypass
+                  moving rules for items.
+                </h4>
```

Comment:
Not sure on the language here - creating settings. I am not too sure about calling usage statuses and units as settings, but struggle to think of a nice alternative. On the backend we the only "setting" we refer to is the spares definition. I did think maybe configuration, but its probably worse for a user and means virtually the same thing.

As a side note I wouldnt use h4 here, best to style the tooltip itself if you want to change the contents/use a typography with variant=h4, as for accessibility the heading order is supposed to be consistent and this is a tooltip not a heading (even if I did before working on accessibility 😄).